TanukiKitTests/ProjectTests.swift

```diff
@@ -20,7 +20,7 @@ class ProjectTests: XCTestCase {
     }
 
     func testGetProjectsEECE() {
-        let session = TanukiKitURLTestSession(expectedURL: "https://code.tiferrei.com/api/v3/projects?page=1&per_page=100&private_token=12345", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
+        let session = TanukiKitURLTestSession(expectedURL: "https://code.tiferrei.com/api/v3/projects?archived=&order_by=created_at&page=1&per_page=20&private_token=12345&search=&simple=&sort=desc&visibility=", expectedHTTPMethod: "GET", jsonFile: "Projects", statusCode: 200)
```

Comment:
`&search=&simple=&sort=desc&visibility=`

This looks very weird. If the parameter name is not given it should not be added to the request. Empty parameter names can leas to unexpected behavior.